probes/codeReviewOneReviewers/impl.go

```diff
@@ -22,23 +22,35 @@ import (
 	"github.com/ossf/scorecard/v4/checker"
 	"github.com/ossf/scorecard/v4/clients"
 	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/internal/probes"
 	"github.com/ossf/scorecard/v4/probes/utils"
 )
 
+func init() {
+	err := probes.Register(probes.Probe{
+		Name:            Probe,
+		Implementation:  Run,
+		RequiredRawData: []probes.CheckName{probes.CodeReview},
+	})
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
This block is effectively identical across all probes. Is it possible to refactor this into:

```go
// probes/util.go or internal/probes.go
func ProbeInit(probe probes.Probe, impl func(raw *checker.RawResults), requiredRawData []probes.CheckName) {
    err := probes.Register(probes.Probe{
		Name:            probe,
		Implementation:  impl,
		RequiredRawData: requiredRawData,
	})
	if err != nil {
		panic(err)
	}
}

// $PROBE/impl.go
func init() {
    probes.ProbeInit(Probe, Run, []probes.CheckName{probes.CodeReview})
}
```

Reply:
Good suggestion. Done in 2ff7302b691d8964b5b0b6088abbdb707d8823cd.